docs/admin/install.rst

```diff
@@ -346,8 +353,8 @@ It is usually a good idea to run Weblate in a separate database, and separate us
 .. hint::
 
    If you don't want to make the Weblate user a superuser in PostgreSQL, you can
-   omit that. In that case you will have to perform some of the migration steps
-   manually as a PostgreSQL superuser in schema Weblate will use:
+   omit that and perform some of the migration steps manually.
+   This is an example of a PostgreSQL superuser in the `schema <https://www.postgresql.org/docs/9.1/ddl-schemas.html>`_ Weblate uses:
```

Comment:
This is now completely wrong. It should be executed as a PostgreSQL superuser. Also, I don't think that the link is useful here. Anybody fiddling with PostgreSQL permissions will understand what a PostgreSQL schema is.

Reply:
This is an example of doing so as a PostgreSQL superuser in the `schema <https://www.postgresql.org/docs/9.1/ddl-schemas.html>`_ Weblate uses

(still haven't read the PostgreSQL documentation) Used it many times though.
Spending effort on trying to figure out what something says isn't conducive to actually doing something with it. If people actually read the documentation, it should. (Even though I mostly don't :)